src/SmartComponents/ImagesTable/ImagesTable.js

```diff
@@ -88,7 +88,8 @@ class ImagesTable extends Component {
                             Create an image
                         </Title>
                         <EmptyStateBody>
-                            Create RHEL OS images for deployment in Amazon Web Services. Images can include
+                            Create OS images for deployment in Amazon Web Services, Azure and
```

Comment:
Maybe use "Microsoft Azure" here, whenever we use the full name we tend to use that.

Reply:
Ah, I meant to change that. Thanks for catching it!

Reply:
Done.